src/wireviz/wv_html.py

```diff
@@ -2,21 +2,27 @@
 # -*- coding: utf-8 -*-
 
 from pathlib import Path
+from typing import List, Union
 import re
 
-from wireviz import __version__, APP_NAME, APP_URL
+from wireviz import __version__, APP_NAME, APP_URL, wv_colors
+from wireviz.DataClasses import Metadata, Options
 from wireviz.wv_helper import flatten2d, open_file_read, open_file_write
 
-def generate_html_output(filename: (str, Path), bom_list):
+def generate_html_output(filename: Union[str, Path], bom_list: List[List[str]], metadata: Metadata, options: Options):
```

Comment:
I haven't included any suggestions here, since this function will hopefully be rebuilt in #74 soon.